CMakeLists.txt

```diff
@@ -647,7 +647,108 @@ ExternalProject_Add(GeographicLib
 # and that config file will tell us everything we need to know about the project's targets.
 set(GeographicLib_DIR ${CMAKE_BINARY_DIR}/GeographicLib-prefix/src/GeographicLib-build/)
 
-#if(BUILD_QT)
+option(BUILD_QT "Build Qt" OFF)
+option(STATIC_QT "Use Static Qt" ON)
+
+if(BUILD_QT)
+
+  if( UNIX )
+    if( STATIC_QT )
+      set(QT_CONFIGURE_COMMAND "-v -release -opensource -static -openssl -no-pch -no-ssse3 -qt-sql-sqlite -plugin-sql-sqlite -skip enginio -no-compile-examples -nomake examples -nomake tests -nomake tools -skip qtmultimedia -confirm-license -prefix ${CMAKE_BINARY_DIR}/Qt-prefix/src/Qt-install")
+    else()
+      set(QT_CONFIGURE_COMMAND "-v -release -opensource -shared -openssl -no-pch -no-ssse3 -qt-sql-sqlite -plugin-sql-sqlite -skip enginio -no-compile-examples -nomake examples -nomake tests -nomake tools -skip qtmultimedia -confirm-license -prefix ${CMAKE_BINARY_DIR}/Qt-prefix/src/Qt-install")
+    endif()
+    ExternalProject_Add(Qt
+      URL http://openstudio-resources.s3.amazonaws.com/dependencies/qt-everywhere-opensource-src-5.6.1.tar.gz
+      URL_MD5 ed16ef2a30c674f91f8615678005d44c
+      CONFIGURE_COMMAND "configure ${QT_CONFIGURE_COMMAND}"
+      PATCH_COMMAND ""
+      BUILD_COMMAND make -j${CPUCOUNT} && make install
+      INSTALL_COMMAND ""
+    )
+  else()
+    # to initialize command prompt run: "C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\vcvarsall.bat" amd64
+    configure_file("${CMAKE_SOURCE_DIR}/dependencies/qt/build_static_qt.bat.in" "${CMAKE_BINARY_DIR}/build_static_qt.bat")
+    configure_file("${CMAKE_SOURCE_DIR}/dependencies/qt/build_shared_qt.bat.in" "${CMAKE_BINARY_DIR}/build_shared_qt.bat")
+    if( STATIC_QT )
+      set(QT_BUILD_SCRIPT "build_static_qt.bat")
+    else()
+      set(QT_BUILD_SCRIPT "build_shared_qt.bat")
+    endif()  
+    ExternalProject_Add(Qt
+      URL http://openstudio-resources.s3.amazonaws.com/dependencies/qt-everywhere-opensource-src-5.6.1.zip
+      URL_MD5 c18a4904d9773894437c10c25f93e50f
+      BUILD_IN_SOURCE 1
+      CONFIGURE_COMMAND ""
+      PATCH_COMMAND ""
+      BUILD_COMMAND cmd /C "${CMAKE_BINARY_DIR}/${QT_BUILD_SCRIPT}"
+      INSTALL_COMMAND ""
+    )
+  endif()     
+  
+else()
+ 
+  # download qt
```

Comment:
Downloading Qt at super-build time, always unpack to Qt-prefix/src/Qt-install so can find it from openstudiocore, no configuration variables are passed from super build to openstudiocore